stable/redis/templates/redis-master-statefulset.yaml

```diff
@@ -39,6 +39,9 @@ spec:
         fsGroup: {{ .Values.master.securityContext.fsGroup }}
         runAsUser: {{ .Values.master.securityContext.runAsUser }}
       {{- end }}
+      {{- if .Values.serviceAccount.create }}
```

Comment:
No need to wrap into an if block.

Reply:
@unguiculus Forget about precedent comment. Fixed in cb669279938dc9f22f48118e0ff239a83d6b6e24